gym/scoreboard/api.py

```diff
@@ -177,8 +177,13 @@ def write_archive(videos, archive_file, env_id=None):
             tar.add(metadata_path, arcname=metadata_name, recursive=False)
 
         # Actually write the manifest file
-        with tempfile.NamedTemporaryFile(mode='w+') as f:
+        # 'delete = False' avoids deleting the file when closing
+        with tempfile.NamedTemporaryFile(mode='w+', delete = False) as f:
```

Comment:
Please match the style of surrounding code (so do `delete=False` rather than `delete = False` here).